target/linux/ipq806x/files-4.14/arch/arm/boot/dts/qcom-ipq8064-qxwlan-e5200.dts

```diff
@@ -0,0 +1,290 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+ *
+ * Copyright (c) 2019 Peng Zhang <sd20@qxwlan.com>
+ *
+*/
+
+#include "qcom-ipq8064-v1.0.dtsi"
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Qxwlan E5200";
+	compatible = "qxwlan,e5200", "qcom,ipq8064";
+
+	memory@0 {
+		reg = <0x42000000 0x1e000000>;
+		device_type = "memory";
+	};
+
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+		rsvd@41200000 {
+			reg = <0x41200000 0x300000>;
+			no-map;
+		};
+	};
+
+	aliases {
+		serial0 = &gsbi4_serial;
+		mdio-gpio0 = &mdio0;
```

Comment:
Usually aliases are reserved for devices that the user interacts with (everything that is labeled, has a pictogram, etc...)

Also wouldn't it be great to wire up status_pass LED to led-boot, led-running? Any maybe status_fail to led-failsafe or use it as a panic-indicator?

Reply:
This should be addressed.